### ISSUE TITLE
Number of fractional digits

### ISSUE THREAD

**Issue by ilazaridis** [bluppfisk/coinprice-indicator#70]
Can we somehow have 8 digits after the decimal point displayed ?

**Comment by bluppfisk** [comment]
Numbers are rounded until what I consider a meaningful amount of digits. For BTC to EUR, for instance, you will merely get 2 decimal places as I cannot fathom that anyone cares how many 1000ths of a Euro their stash is worth. But for coins of a smaller value in fiat, you'll see many more decimal places automatically (up to 8 IIRC).

**Comment by colourful-land** [comment]
Problem is, the meaningful number is different for different tokens.

the default asset loaded is the value of ETH in terms of BTC. It shows 0.027 currently.

This information isn't very helpful, since it barely moves for a long time. Users would appreciate precision like 0.0278.

Didn't find any config options to change the precision.

**Comment by bluppfisk** [comment]
commit 9536ddd should address this issue. It allows you to adjust the maximum number of decimals as well as the number of significant digits in user.conf. I also set a new default of 3 significant digits. Let me know how you fare with this change :)

**Comment by colourful-land** [comment]
Thanks but … I hope @ilazaridis would agree, that when a user is using an indicator app, he actually wasn't expecting that the settings are changed by editing user.conf. examples are fcitx (Chinese input engine), system monitor apps and flameshot. but this is better than not able to change the decimal points for sure.

**Comment by bluppfisk** [comment]
Fair enough... This is a GUI app after all. I should include a GUI for the settings. State management is horrible/absent at the moment in this application, so I'd rather set this globally than on a per-ticker basis. It will also likely not apply until the next tick comes in.

**Comment by Mohammad-Haseeb** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @LouiseLog, @yangsoon, @old101, @chrysos349, @tlvenn, @robinm8, @Piterden, @Twiggy547, @RonWalker22, @blockconsortium, @cseder, @ognjenstrbanovic, @DannyBen, @chenziya, @EzVoodoo, @wkirgsn, @xyNNN, @HiKWang, @dimtoneff, @rancid
